heartbeat/pgsql

```diff
@@ -593,7 +597,7 @@ pgsql_real_start() {
     pgctl_options="$pgctl_options -o '$postgres_options'"
 
     # Invoke pg_ctl
-    runasowner "unset PGUSER; unset PGPASSWORD; $OCF_RESKEY_pgctl $pgctl_options start"
+    runasowner "unset PGUSER; unset PGPASSWORD; $OCF_RESKEY_pgctl $pgctl_options -W start"
```

Comment:
Is the -W flag backwards compatible?

Reply:
Yes, it is for backwards compatible.
The default action of "pg_ctl start" and "pg_ctl promote" is changed on PostgreSQL 10.
The default is
PostgreSQL 9.x : no wait (-W)
PostgreSQL 10 : wait (-w)

Reply:
Makes sense, thank you!